app/endpoints.js

```diff
@@ -48,6 +48,7 @@ const endpoints = {
     putSupplier: options => `${orderApiUrl}/api/v1/orders/${options.orderId}/sections/supplier`,
     putCatalogueSolutions: options => `${orderApiUrl}/api/v1/orders/${options.orderId}/sections/catalogue-solutions`,
     putAdditionalServices: options => `${orderApiUrl}/api/v1/orders/${options.orderId}/sections/additional-services`,
+    putAssociatedServices: options => `${orderApiUrl}/api/v1/orders/${options.orderId}/sections/associated-services`,
```

Comment:
Can you add this to the api helper instead please. No more adding this this guy

Reply:
👍